src/crypto/CryptoInterface.c

```diff
@@ -34,49 +34,66 @@
 /**
  * Struct to hold private key information
  */
-typedef struct BSLB_CryptoKey_s
+typedef struct BSL_CryptoKey_s
 {
     /// Pointer to OpenSSL PKEY struct (used in hmac ctx)
     EVP_PKEY *pkey;
     /// Pointer to raw key information (used in cipher ctx)
     BSL_Data_t raw;
-} BSLB_CryptoKey_t;
 
-static int BSLB_CryptoKey_Deinit(BSLB_CryptoKey_t *key)
+    BSL_Crypto_KeyStats_t stats;
+} BSL_CryptoKey_t;
+
+static int BSL_CryptoKey_Init(BSL_CryptoKey_t *key)
+{
+    for (uint64_t i = 0; i < BSL_CRYPTO_KEYSTATS_MAX_INDEX; i++)
+    {
+        key->stats.stats[i] = 0;
+    }
+
+    return 0;
+}
+
+static int BSL_CryptoKey_Deinit(BSL_CryptoKey_t *key)
 {
     fflush(stdout); // NOLINT
     EVP_PKEY_free(key->pkey);
     BSL_Data_Deinit(&(key->raw));
+
+    for (uint64_t i = 0; i < BSL_CRYPTO_KEYSTATS_MAX_INDEX; i++)
+    {
+        key->stats.stats[i] = 0;
+    }
     return 0;
 }
 
 // NOLINTBEGIN
 /// @cond Doxygen_Suppress
-#define M_OPL_BSLB_CryptoKey_t() M_OPEXTEND(M_POD_OPLIST, CLEAR(API_2(BSLB_CryptoKey_Deinit)))
+#define M_OPL_BSL_CryptoKey_t() M_OPEXTEND(M_POD_OPLIST, CLEAR(API_2(BSL_CryptoKey_Deinit)))
```

Comment:
I know this was pre-existing, but is it actually okay to extend POD_OPLIST? It seems like the `EVP_PKEY` and the `BSL_Data_t` will not necessarily be managed properly.

If this is too messy to deal with now and it's not guaranteed by the public API, please add a sustainment ticket about this.